scripts/splunk-default-config

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# no metrics
+export OTEL_METRICS_EXPORTER=none
+
+# otlp traces exporter
+export OTEL_TRACES_EXPORTER="${OTEL_TRACES_EXPORTER:-otlp}"
+# otlp-grpc protocol
+export OTEL_EXPORTER_OTLP_TRACES_PROTOCOL="${OTEL_EXPORTER_OTLP_TRACES_PROTOCOL:-grpc}"
+
+# tracecontext, baggage propagation
+export OTEL_PROPAGATORS="${OTEL_PROPAGATORS:-tracecontext,baggage}"
+
+# no sampling
+export OTEL_TRACES_SAMPLER="${OTEL_TRACES_SAMPLER:-always_on}"
+
+# splunk token header if OLTP
+if [ -n "$SPLUNK_ACCESS_TOKEN" ] && [ "$OTEL_TRACES_EXPORTER" = "otlp" ]
+then
+    HEADER="X-SF-TOKEN=$SPLUNK_ACCESS_TOKEN,"
```

Comment:
If `OTEL_EXPORTER_OTLP_HEADERS` isn't set, will this result in a trailing comma and is that ok?